src/main/java/uk/ac/ebi/pride/AppContext.java

```diff
@@ -207,6 +259,9 @@ public synchronized void addDataFile(DataFile dataFile) {
 
             firePropertyChange(ADD_NEW_DATA_FILE, null, dataFile);
         }
+        if(isResubmission){
+            getResubmissionRecord().getResubmission().getResubmission().put(dataFile, ResubmissionFileChangeState.ADD);
```

Comment:
can we change doundle getResubmission functions to something unique funtion names